api/server.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+/*
+|--------------------------------------------------------------------------
+| Http server
+|--------------------------------------------------------------------------
+|
+| This file bootstrap Adonisjs to start the HTTP server. You are free to
+| customize the process of booting the http server.
+|
+| """ Loading ace commands """
+|     At times you may want to load ace commands when starting the HTTP server.
+|     Same can be done by chaining `loadCommands()` method after
+|
+| """ Preloading files """
+|     Also you can preload files by calling `preLoad('path/to/file')` method.
+|     Make sure to pass relative path from the project root.
+*/
+
+const path = require('path');
+
+const { Ignitor } = require('@adonisjs/ignitor');
+
+const config = require('../src/config/config').globalConfig;
+
+process.env.HOST = '0.0.0.0';
+process.env.PORT = config.port;
```

Comment:
Those should come from the `.env` file but there won't be one in this project...